src/logic/map_objects/tribes/building.cc

```diff
@@ -413,6 +413,8 @@ derived class' init.
 ===============
 */
 bool Building::init(EditorGameBase& egbase) {
+	egbase.mutex_lock();
```

Comment:
Uhhhm,  better make this a Ctor/DTor pattern like
{
   Lock egLock = egbase.mutex_lock();

} // Dtor will call unlock

If I jump out og such a function wit return, the lock will stick...